File: final-evaluation-generated-code/test-2/deterministic.js
import fetch from 'node-fetch';

const URL = "http://localhost:3000/http-express-calculator-simple/actions/subtract";

const bodyInput = 64342111.59;

async function invokeactionSubtract(inputValue) {
    try {
        const response = await fetch(URL, {
            method: "POST",
            headers: {
                "Content-Type": "application/json",
            },
            body: JSON.stringify(inputValue),
        });

        if (!response.ok) {
            throw new Error(`Failed to invokeaction subtract: ${response.status} ${response.statusText}`);
        }

        const subtract = await response.json();
        console.log("subtract value:", subtract);

        return subtract;
    } catch (error) {
        console.error("Error:", error);
    }
}

invokeactionSubtract(bodyInput);

File: final-evaluation-generated-code/test-8/deterministic.py
import requests

URL = "http://localhost:3000/http-express-calculator-simple/actions/subtract"

payload = -54004888.43

def invokeaction_subtract(payload_value):
  try:
    response = requests.post(URL,json=payload_value, headers={"Content-Type": "application/json"})
    if response.status_code == 200:
      subtract = response.json()
      print("subtract value:", subtract)
      return subtract
    else:
      print(f"Failed to invokeaction subtract. Status code: {response.status_code}, Response: {response.text}")

  except requests.exceptions.RequestException as error:
    print("Error during the request:", error)

invokeaction_subtract(payload)